rocky/account/forms/account_setup.py

```diff
@@ -159,7 +159,7 @@ def save(self, **kwargs):
                 status=OrganizationMember.STATUSES.ACTIVE,
             )
             member.groups.add(selected_group.id)
-            if member.is_admin or self.user.is_superuser:
+            if member.has_perm("change_organizationmember") or self.user.is_superuser:
```

Comment:
Why does a new member that can change organization members automatically get scan level 4? Wouldn't it be better to have the user set this explicitly? And if we want to have this way, shouldn't we this and probably make this also clear in the user interface?

This is something we can change in a later PR.

Reply:
Here admins have by default trusted and acknowledged clearance levels by L4 so they can start scans but do not have clearance permission on objects, so redteamers are the only ones that can increase clearance levels on objects. 

We can refactor this part to put defaults at member creation when member is an admin.